lib/payload/scanner/SSL_TLS/engine.py

```diff
@@ -0,0 +1,674 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Pradeep Jairamani (github.com/pradeepjairamani)
+
+import os, time
+import socket
+import struct
+import re
+from sets import Set
+
+class TLS:
+    def __init__(self, host = "", port = 0):
```

Comment:
@pradeepjairamani, there should be no space between `host = ""` and `port = 0`. It should be `host="", port=0` according to PEP-8.